Patches/The GiantRace/ThingDefs_Misc/GiantWeapons_Ranged.xml

```diff
@@ -215,7 +262,7 @@
 				defName="GI_Scrapmetal" or
 				defName="GI_Rockpillar_granite" or
 				defName="GI_Rockpillar_Marble"
-				]/recipeMaker</xpath>
+				]//recipeMaker</xpath>

```

Comment:
Is this intentional? What's the purpose?

Reply:
Was part of earlier stuff trying to get it to work and didn't cause errors once it did work so I forgot about it. I've removed the double slash since it isn't needed.